### PR TITLE
[mono] More exception tests

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

**Comment by lewurm** [comment]
does it make a difference on bytecode level?

File: mono/mini/exceptions.cs [hunk 1]
```diff
@@ -1683,6 +1683,19 @@ class Tests
 		if (failed)
 			return 4;
 
+		try {
```
Comment (vargaz) [inline mono/mini/exceptions.cs]: Whats the difference with the previous test ?

**Comment by lewurm** [comment]
why was this PR closed?  shouldn't we merge it in order to test ttps://github.com/mono/mono/pull/5900 ?

**Comment by vargaz** [comment]
The test was a duplicate.

**Pull request closed without merging**